File: ShippingCompanyServices/ShippingCompanyServices/Models/ShippingCompany.cs
using System.ComponentModel.DataAnnotations;

namespace ShippingCompanyServices.Models
{
    public class ShippingCompany
    {
        [Required]
        public int ID { get; set; }
        public string Name { get; set; }
        public string Password { get; set; }
        public string Geography { get; set; }
    }
}

File: ShippingService/src/main/java/com/example/shippingservice/ShippingCompany/shippingCompany.java
package com.example.shippingservice.ShippingCompany;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Entity
@Table(name = "shippingcompany")
@AllArgsConstructor
@NoArgsConstructor
@Setter
@Getter
public class shippingCompany {
    @Id
    private String name;

    private String password;
    private String geography;

}
